src/game/logic/object/object.cpp

```diff
@@ -202,3 +203,87 @@ void Object::Clear_Model_Condition_State(ModelConditionFlagType a)
         m_drawable->Clear_Model_Condition_State(a);
     }
 }
+
+void Object::Attempt_Damage(DamageInfo *info)
+{
+#ifdef GAME_DLL
+    Call_Method<void, Object, DamageInfo *>(PICK_ADDRESS(0x00547DA0, 0x007D134E), this, info);
+#endif
+}
+
+void Object::Clear_Disabled(DisabledType type)
+{
+#ifdef GAME_DLL
+    Call_Method<void, Object, DisabledType>(PICK_ADDRESS(0x005489F0, 0x007D1FE8), this, type);
+#endif
+}
+
+bool Object::Can_Crush_Or_Squish(Object *obj)
+{
+#ifdef GAME_DLL
+    return Call_Method<bool, Object, Object *>(PICK_ADDRESS(0x005471D0, 0x007D0486), this, obj);
+#else
+    return false;
+#endif
+}
+
+void Object::Defect(Team *team, unsigned int i)
+{
+#ifdef GAME_DLL
+    Call_Method<void, Object, Team *, unsigned int>(PICK_ADDRESS(0x0054EEC0, 0x007D924B), this, team, i);
+#endif
+}
+
+unsigned char Object::Get_Crushable_Level() const
+{
+    return Get_Template()->Get_Crushable_Level();
+}
+
+unsigned char Object::Get_Crusher_Level() const
+{
+    return Get_Template()->Get_Crusher_Level();
+}
+
+void Object::Kill(DamageType damage, DeathType death)
+{
+#ifdef GAME_DLL
+    Call_Method<void, Object, DamageType, DeathType>(PICK_ADDRESS(0x00548300, 0x007D1733), this, damage, death);
+#endif
+}
+
+void Object::On_Collide(Object *other, const Coord3D *loc, const Coord3D *normal)
+{
+    for (BehaviorModule **module = m_allModules; *module != nullptr; module++) {
+        CollideModuleInterface *collide = (*module)->Get_Collide();
+
+        if (collide != nullptr) {
+            if (Get_Status_Bits().Test(OBJECT_STATUS_NO_COLLISIONS)) {
```

Comment:
was break i guess rather?

Reply:
IDA pseudo showed return. And return seems to fit.